### DISCUSSION TITLE
Customisation of fields with children

### DISCUSSION THREAD

**Discussion by plushdohn** [payloadcms/payload#2411] [Feature Requests & Ideas]
# Documentation Issue

I think the docs should have an example on how to customise the `Field` components in field types such as `array`, `blocks` or `group`. Right now overriding the `Field` component in any of those stops rendering its children, which is fine, but it should also be explained how to continue rendering their children using Payload's default components, say in the case I want to customise a Block's `RowLabel` component, while keeping the rendering logic for children the same.

**Comment by jmikrut** [comment]
Hey @plushdohn — did you notice that you can override block and array fields' `RowLabel` component without replacing the whole field component itself?

https://payloadcms.com/docs/fields/array#admin-config

Because replacing a group, array, or block field itself is going to be a bit more complex than what the docs might cover, I think this is a good candidate for a future tutorial / example repo.

I will tag this as a feature request and we'll get a full example added. 👍

**Reply by plushdohn** [reply]
Thanks for your help! Although I do not see the option to override `RowLabel` in blocks, am I missing something?
